app/javascript/dashboard/routes/dashboard/contacts/components/AddReminder.vue

```diff
@@ -0,0 +1,146 @@
+<template>
+  <div class="wrap">
+    <div class="input-select-wrap">
+      <textarea
+        v-model="content"
+        class="input--reminder"
+        @keydown.enter.shift.exact="onAdd"
+      >
+      </textarea>
+      <div class="select-wrap">
+        <div class="select">
+          <div class="date-wrap">
```

Comment:
Can we use this? https://get.foundation/sites/docs/forms.html#inline-labels-and-buttons